foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -43,10 +43,14 @@ Back in the `index.html` lets add the `<html>` element by typing out its opening
 
 ~~~html
 <!DOCTYPE html>
-<html>
+<html lang="en">
 </html>
 ~~~
 
+#### What is the `lang` attribute? 
+
+`lang` specifies the language of the text content in that element. This attribute is primarily used for boosting accessibility of the webpage. It allows assistive technologies, for example screen readers, to adapt according to the language and invoke correct pronunciation.
```

Comment:
Maybe "improving accessibility" instead of "boosting"?

Reply:
Yeah, that sounds better